documentation/LABEL_FILE_FORMAT.md

```diff
@@ -0,0 +1,78 @@
+# DeepCell Label File Format
+
+## Supported input files
+
+To create a project with DeepCell Label, POST a request to `label.deepcell.org/api/project` with the following form data:
+
+- `images`: a URL that downloads
+  - a .tiff
+  - a .png
+  - an .npz, or zipped numpy array
```

Comment:
Are there any assumptions about the keys in the numpy file, e.g. X and y?

Reply:
added note about X and y arrays, which are used for images/segmentation respectively if present, and otherwise the first array in the .npz is used instead